projects/ngrx.io/content/guide/store/testing.md

```diff
@@ -75,7 +75,7 @@ In this example based on the [walkthrough](guide/store/walkthrough), we mock the
 
 <div class="alert is-helpful">
 
-**Note:** `MockStore` will reset all of the mocked selectors after each test (in the `afterEach()` hook) by calling the `MockStore.resetSelectors()` method.
+**Note:** `MockStore` will **not** reset mocked selectors after each test. You can reset selectors by calling the `MockStore.resetSelectors()` method in `afterEach()` hook.
```

Comment:
I think we can remove this alert, and show how you can reset the store manually after each test.
About the re-added tests. I think we're already testing that  `resetSelector` does what it's supposed to do, so these are redundant and can be removed. 
Thoughts?

Reply:
Ok, that sounds good to me. I was not sure about this test.
I'll add a code example in the doc